src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -104,6 +104,15 @@ public sealed override bool IgnoresAccessibility
             }
         }
 
+        // PROTOTYPE(ndsm): it feels strange to have both 'DisableNullableAnalysis' and 'IsNullableAnalysisEnabled' members. But maybe it's fine.
```

Comment:
Maybe `IsNullableAnalysisEnabledCore` could be named something like `CanNullableAnalysisBeEnabled` so it's clear that it's not the final thing, but must be combined with the `DisableNullableAnalysis` flag.